features/custom-fonts.js

```diff
@@ -5,7 +5,7 @@ style.innerHTML = `
 @import url('https://fonts.googleapis.com/css2?family=${name.replaceAll(')', '').replaceAll("'", '').replaceAll('}', '').replaceAll(' ', '+')}:wght@200;300;400;500;600;700&display=swap');
 
 * {
-    font-family: '${name}', sans-serif;
+    font-family: '${name.replaceAll('\\', '').replaceAll("'", '').replaceAll('}', '')}', sans-serif;
```

Comment:
i told you about this forever ago bro

Reply:
there were like 3 different things we talked about with this, I don't remember all of them